.github/workflows/zeebe-ci.yml

```diff
@@ -445,9 +443,6 @@ jobs:
         with:
           # fixed to avoid triggering false positive; see https://github.com/golangci/golangci-lint-action/issues/535
           version: v1.55.2
-          # caching issues, see: https://github.com/golangci/golangci-lint-action/issues/244#issuecomment-1052190775
-          skip-pkg-cache: true
-          skip-build-cache: true
```

Comment:
I removed those parameters since the more recent versions of the action do not have it anymore, see https://github.com/golangci/golangci-lint-action/releases/tag/v5.0.0

@npepinpe do you know if that will be ok or another solution is needed?

Reply:
According to the action repository, [this has been fixed](https://github.com/golangci/golangci-lint-action/issues/135), so I think it should be fine.